src/ZipStreamFacade.php

```diff
@@ -5,7 +5,9 @@
 use Illuminate\Support\Facades\Facade;
 
 /**
- * @see \Stechstudio\LaravelZipstream\Skeleton\SkeletonClass
+ * @method static \STD\ZipStream\ZipStream create(?string $name = null, array $files = [])
```

Comment:
should be `\STS\ZipStream` not `\STD` right?

Reply:
Lol, you're totally right.